src/main/java/cromwell/binding/DiskType.java

```diff
@@ -0,0 +1,15 @@
+package cromwell.binding;
+
+public enum DiskType {
```

Comment:
agree on the uselessness of enumeration. only asked as when i made a quick skim I didn't see any Enum-ness being used but emphasis on quick skim :)